paparazzi-gradle-plugin/src/test/projects/configuration-cache-generated-sources/build.gradle

```diff
@@ -0,0 +1,40 @@
+plugins {
+  id 'com.android.library'
+  id 'kotlin-android'
+  id 'app.cash.paparazzi'
+}
+
+android {
+  namespace 'app.cash.paparazzi.plugin.test'
+  compileSdk libs.versions.compileSdk.get() as int
+  defaultConfig {
+    minSdk libs.versions.minSdk.get() as int
+  }
+  compileOptions {
+    sourceCompatibility = libs.versions.javaTarget.get()
+    targetCompatibility = libs.versions.javaTarget.get()
+  }
+  kotlinOptions {
+    jvmTarget = libs.versions.javaTarget.get()
+  }
+}
+
+androidComponents {
+  onVariants(selector().all()) { variant ->
+    def sourceGen =
+      project.tasks.register("do${variant.name.capitalize()}SourceGen", SourceGenTask) {
+        it.outputDirectory.set(null)
```

Comment:
Why do you explicitly clear it? AGP will overwrite it regardless of current value, right?

Reply:
ah, leftover from earlier debugging.  removed.